tests/test_api_client.py

```diff
@@ -0,0 +1,38 @@
+from functools import partial
+
+import pytest
+
+from rossum.lib.api_client import RossumClient
+from tests.conftest import INBOXES_URL, match_uploaded_json, QUEUES_URL
+
+
+@pytest.mark.usefixtures("rossum_credentials")
+class TestRossumClient:
+    rossum_client = RossumClient(None)
+
+    @pytest.mark.usefixtures("mock_login_request")
+    def test_create_inbox(self, requests_mock):
```

Comment:
testing also the case with email and without email_prefix would be nice and would possibly solve the failing pipeline  - but I am not sure how to do it right - It would be tempting to use @pytest.mark.parametrize - but to be honest the code is already quite complex and adding that would make it really challenging to read - so after all, I wouldn't mind to leave it like it is and maybe break code coverage requirement for this one

Reply:
I agree that parametrization in this case may get ugly.
We should probably make pytest coverage happier, though.
So I have added tests for the untested scenarios.